components/datap_json/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "icu-datap-json"
+description = "ICU4X data provider that reads from JSON files"
+version = "0.0.1"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+readme = "README.md"
+repository = "https://github.com/unicode-org/icu4x"
+license = "MIT/Apache-2.0"
```

Comment:
Instead of `license`, this should be `license-file` pointing to the top-level `LICENSE` file.

Reply:
I copied this from the other Cargo.toml files (e.g. components/icu4x/Cargo.toml).  Should those other files be changed, too?

Reply:
Consensus: check this in as-is, and fix all the Cargo.toml together.

Reply:
Follow-up: #114